backend/vaa-strapi/.env.example

```diff
@@ -3,19 +3,19 @@
 ## Use the .env file in the project root when using Docker
 
 # Backend dependencies
-STRAPI_HOST=0.0.0.0
+STRAPI_HOST=localhost
 STRAPI_PORT=1337
 APP_KEYS="toBeModified1,toBeModified2"
 API_TOKEN_SALT=tobemodified
 ADMIN_JWT_SECRET=tobemodified
-JWT_SECRET=tobemodified
-DATABASE_HOST=postgres
+JWT_SECRET=tobemodified2
+DATABASE_HOST=localhost
```

Comment:
This needs to be `localhost` in order for the Docker service to be able to access a DB service which is run outside of its own Docker compose stack (which is exactly what we need if we spin up only the backend service with this local `.env` file).